.drone.yml

```diff
@@ -210,6 +210,6 @@ trigger:
     - pull_request
 ---
 kind: signature
-hmac: 83134bebe48b9c0fb9ff12b8ef0fa324462208ae9802d027fc6e169c36ceac89
+hmac: c448ed8c5e80617ecaaf250543ad05b0ca2fc1e604656dc7593eb7f70a91ba92
```

Comment:
Reminder to self to also sign the drone.yml again after updating docker tag to final version prior to merge